docs/migrating-configs.md

```diff
@@ -157,6 +157,99 @@ IP addresses and domain names can also include a port number (`1.2.3.4:80`).
   }
 }
 ```
+## From Version 3.1.0 to 3.2.0
+
+There are not any breaking changes between versions 3.1.0 and 3.2.0 of the configuration specification. Any valid 3.1.0 configuration can be updated to a 3.2.0 configuration by simply changing the version string in the config.
+
+The following is a list of notable new features, deprecations, and changes.
+
+### Partition Resizing
+
+The `partition` section gained a new `resize` field. When specified Ignition will resize an existing partition if it matches the config in all respects except the partition size. Refer to the [operator-notes][operator-notes] for more information.
```

Comment:
"When `true`,"
`Refer to the [operator notes]`.  But is there anything in the operator notes that we haven't just said here?  We could just drop the last sentence.